rpc/handlers.go

```diff
@@ -996,3 +1000,26 @@ func (h *Handler) AddDeclareTransaction(declareTx json.RawMessage) (*DeclareTxRe
 func (h *Handler) Version() (string, *jsonrpc.Error) {
 	return h.version, nil
 }
+
+func (h *Handler) TransactionStatus(hash felt.Felt) (Status, *jsonrpc.Error) {
+	var status Status
+
+	receipt, txErr := h.TransactionReceiptByHash(hash)
+	if txErr == nil {
+		status = receipt.Status
+	} else if txErr == ErrTxnHashNotFound {
```

Comment:
let's use `errors.Is`

Reply:
the problem that `ErrTxnHashNotFound` is not an error but `&jsonrpc.Error{Code: 25, Message: "Transaction hash not found"}`

Reply:
Ah, my bad